src/components/InputAnt/InputAnt.jsx

```diff
@@ -0,0 +1,5 @@
+import React from "react";
+import { Input } from "antd";
+import "./style.css";
+
+export const InputAnt = () => <Input className="input" />;
```

Comment:
I would suggest to rename it otherwise - AntInput imported one, plain Input - internal one. To rename on import you can do
`import { Input as AntInput } from 'antd'